src/resources/image-roll_ui.glade

```diff
@@ -70,6 +70,21 @@ Author: Robert Węcławski
         <property name="can-focus">False</property>
         <property name="orientation">vertical</property>
         <property name="homogeneous">True</property>
+        <child>
+          <object class="GtkButton" id="copy_menu_button">
+            <property name="label" translatable="yes">Copy</property>
+            <property name="visible">True</property>
+            <property name="can-focus">True</property>
+            <property name="receives-default">True</property>
+            <property name="relief">none</property>
+            <accelerator key="s" signal="clicked" modifiers="GDK_SHIFT_MASK | GDK_CONTROL_MASK"/>
```

Comment:
Save as has this accelerator, since you have added shortcut in connect_keybinds I think you can remove the accelerator here and the one for save current image